roles/custom/matrix-bridge-mautrix-wsproxy/templates/systemd/matrix-mautrix-wsproxy.service.j2

```diff
@@ -24,8 +24,10 @@ ExecStart={{ devture_systemd_docker_base_host_command_docker }} run --rm --name
 			--user={{ matrix_user_uid }}:{{ matrix_user_gid }} \
 			--cap-drop=ALL \
 			--network={{ matrix_docker_network }} \
-			-v {{ matrix_mautrix_wsproxy_config_path }}:/data:z \
+			{% if matrix_mautrix_wsproxy_bind_port %}
 			-p {{ matrix_mautrix_wsproxy_port }}:29331 \
+			{% endif %}
+			--mount type=bind,src={{ matrix_mautrix_wsproxy_config_path }},dst=/data:z \
```

Comment:
The `:z` part is incorrect syntax, I believe. Consider removing it.

Reply:
Sorry, I committed without running. I didn't finish the container labels yet, but will do that and have it all in a working state soon, hopefully tonight already 🤞